src/signing.rs

```diff
@@ -113,6 +117,19 @@ mod feature_gated {
             Ok(Signature { v, r, s })
         }
 
+        fn sign_message(&self, message: &[u8]) -> Result<Signature, SigningError> {
+            let message = Message::from_slice(&message).map_err(|_| SigningError::InvalidMessage)?;
+            let (recovery_id, signature) = Secp256k1::signing_only()
```

Comment:
Can you use the newly introduced shared `CONTEXT` here?

Reply:
Uses the new context now.